spec/beaker-hostgenerator/parser_spec.rb

```diff
@@ -113,16 +113,23 @@ module BeakerHostGenerator
       end
 
       context 'When using arbitrary host settings' do
+        it 'Properly tokenize settings' do
```

Comment:
We should be specific about supporting lists in this description rather than just referring them as general settings, so maybe throw in the word "list" or "array" somewhere in here?